website/content/ja/contribute/projects/translations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 翻訳
```

Comment:
cc maintainers

In some other CNCF projects and in k/website, "Localization" is used instead of "Translation" Do you think it would be better to update it to match upstream?